### DISCUSSION TITLE
How can I pass arguments when using Doctr API?

### DISCUSSION THREAD

**Discussion by AASHISHAG** [mindee/doctr#1416] [Q&A]
```
import requests
with open('/path/to/your/doc.jpg', 'rb') as f:
    data = f.read()
response = requests.post("http://localhost:8002/ocr", files={'file': data}).json()
```

Example: `resolve_blocks=False`

**Comment by felixdittrich92** [comment]
Hi @AASHISHAG :wave:,

the api code we provide is very basic as starting point you can use this as a template and configure to your needs :)
At the end it's a simple fastapi RestAPI
